### PR TITLE
API-9877: lookup endpoints

### PR DESCRIPTION
**Pull request by grz-gajda**
# 🚀 Links

- [Jira](https://livechatinc.atlassian.net/browse/API-9860)

# 📓 Description

Two new endpoints in _configuration-api_: `get_license_id` and `get_organization_id`.

### CODE REVIEW

File: src/pages/management/configuration-api/v3.4/index.mdx [hunk 1]
```diff
@@ -2823,6 +2823,96 @@ curl -X POST \
...
+#### Specifics
+
+|                  |                                                                        |
+| ---------------- | ---------------------------------------------------------------------- |
+| HTTP Method      | **GET**                                                                |
+| Method URL       | `https://api.livechatinc.com/v3.4/configuration/action/get_license_id` |
+
+#### Request
+
+| Parameter         | Required | Data type | Notes |
```
Comment (ale-kacperczyk) [inline src/pages/management/configuration-api/v3.4/index.mdx:2853]:
```suggestion
| Query string         | Required | Data type | Notes |
```

File: src/pages/management/configuration-api/v3.4/index.mdx [hunk 2]
```diff
@@ -2823,6 +2823,96 @@ curl -X POST \
...
+#### Specifics
+
+|                  |                                                                             |
+| ---------------- | --------------------------------------------------------------------------- |
+| HTTP Method      | **GET**                                                                     |
+| Method URL       | `https://api.livechatinc.com/v3.4/configuration/action/get_organization_id` |
+
+#### Request
+
+| Parameter         | Required | Data type | Notes  |
```
Comment (ale-kacperczyk) [inline src/pages/management/configuration-api/v3.4/index.mdx:2893]:
```suggestion
| Query string         | Required | Data type | Notes  |
```

File: src/pages/management/configuration-api/v3.4/index.mdx [hunk 3]
```diff
@@ -2823,6 +2823,96 @@ curl -X POST \
...
+| ----------- | ---------------------------------------------------------------- |
+| `GET`       | `https://api.livechatinc.com/v3.4/configuration/action/<action>` |
+
+<Section>
+
+<Text>
+
+### Get License ID
+
+Returns license ID for given organization ID.
```
[the remark this answers is not in the document]
Comment (grz-gajda) [reply src/pages/management/configuration-api/v3.4/index.mdx]: Corrected.

File: src/pages/management/configuration-api/v3.4/index.mdx [hunk 4]
```diff
@@ -2823,6 +2823,96 @@ curl -X POST \
...
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Get Organization ID
+
+Returns organization ID for given license ID.
```
[the remark this answers is not in the document]
Comment (grz-gajda) [reply src/pages/management/configuration-api/v3.4/index.mdx]: Corrected.

File: src/pages/management/configuration-api/v3.4/index.mdx (hunk 1 above)
Comment (grz-gajda) [reply src/pages/management/configuration-api/v3.4/index.mdx]: Corrected.

File: src/pages/management/configuration-api/v3.4/index.mdx (hunk 2 above)
Comment (grz-gajda) [reply src/pages/management/configuration-api/v3.4/index.mdx]: Corrected.

Comment (grz-gajda) [reply src/pages/management/configuration-api/v3.4/index.mdx]: Corrected.

File: src/pages/management/configuration-api/v3.4/index.mdx [hunk 5]
```diff
@@ -2823,6 +2823,96 @@ curl -X POST \
 
 To see the list of available webhooks v3.4, [visit the Webhooks](/management/webhooks/v3.4/) document.
 
+# Lookup
```
[the remark this answers is not in the document]
Comment (grz-gajda) [reply src/pages/management/configuration-api/v3.4/index.mdx:2826]: What do you think about _Organizations_? As lookup methods are part of _Organization_ entity.

[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply src/pages/management/configuration-api/v3.4/index.mdx:2826]: I think it's better, although it makes me think of Global Accounts (as if the endpoint should belong to GAcc API). What does @bkubiak think about the naming in this case?

[the remark this answers is not in the document]
Comment (bkubiak) [reply src/pages/management/configuration-api/v3.4/index.mdx:2826]: Hmm, I don't know to be honest. I'm not sure if organizations or lookup are good names.

How about `helpers` or `other actions`? 😄

File: payloads/management/v3.4/configuration-api/responses/lookup/getLicenseID.json [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{
+  "license_id": 10000000
```
[the remark this answers is not in the document]
Comment (bkubiak) [reply payloads/management/v3.4/configuration-api/responses/lookup/getLicenseID.json:2]: Unless we don't follow the rule in conf-api docs? @ale-kacperczyk do you know if that's the case?

File: src/pages/management/configuration-api/v3.4/index.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply src/pages/management/configuration-api/v3.4/index.mdx:2826]: Do you plan to add any other methods to this category in the near future? Maybe that could help use come up with a name

File: payloads/management/v3.4/configuration-api/responses/lookup/getOrganizationID.json [hunk 7]
```diff
@@ -0,0 +1,3 @@
+{
+  "organization_id": "142cf3ad-5d54-4cf6-8ce1-3773d14d7f3f"
```
[the remark this answers is not in the document]
Comment (grz-gajda) [reply payloads/management/v3.4/configuration-api/responses/lookup/getOrganizationID.json:2]: We don't have default organization ID for docs in README.md, how about I add `142cf3ad-5d54-4cf6-8ce1-3773d14d7f3f` there?

File: src/pages/management/configuration-api/v3.4/index.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (grz-gajda) [reply src/pages/management/configuration-api/v3.4/index.mdx]: What about `Other`? We use it in the documentation for Agent API.

[the remark this answers is not in the document]
Comment (bkubiak) [reply src/pages/management/configuration-api/v3.4/index.mdx]: @ale-kacperczyk currently, there are no plans for extending such methods further

File: payloads/management/v3.4/configuration-api/responses/lookup/getOrganizationID.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (bkubiak) [reply payloads/management/v3.4/configuration-api/responses/lookup/getOrganizationID.json:2]: ok

[the remark this answers is not in the document]
Comment (bkubiak) [reply payloads/management/v3.4/configuration-api/responses/lookup/getOrganizationID.json:2]: And perhaps this should be moved to the conflu?

File: src/pages/management/configuration-api/v3.4/index.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (ale-kacperczyk) [reply src/pages/management/configuration-api/v3.4/index.mdx]: OK, I'd stick to Other. Consistency with other APIs is a good argument :)

File: payloads/management/v3.4/configuration-api/index.js [hunk 8]
```diff
@@ -19,3 +19,6 @@ export { default as listWebhooks } from "./responses/webhooks/listWebhooks.json"
 export { default as listWebhookNames } from "./responses/webhooks/listWebhookNames.json"
 
 export { default as getWebhooksState } from "./responses/webhooks/getWebhooksState.json"
+
+export { default as getOrganizationID } from "./responses/lookup/getOrganizationID.json"
```
Comment (ale-kacperczyk) [inline payloads/management/v3.4/configuration-api/index.js:23]:
```suggestion
export { default as getOrganizationID } from "./responses/other/getOrganizationID.json"
```

File: payloads/management/v3.4/configuration-api/index.js [hunk 9]
```diff
@@ -19,3 +19,6 @@ export { default as listWebhooks } from "./responses/webhooks/listWebhooks.json"
...
+export { default as getLicenseID } from "./responses/lookup/getLicenseID.json"
```
Comment (ale-kacperczyk) [inline payloads/management/v3.4/configuration-api/index.js:24]:
```suggestion
export { default as getLicenseID } from "./responses/other/getLicenseID.json"
```

Comment (ale-kacperczyk) [inline payloads/management/v3.4/configuration-api/index.js:24]:
```suggestion
export { default as getLicenseID } from "./responses/other/getLicenseID.json"
```

**Comment by grz-gajda** [comment]
@ale-kacperczyk path to JSON files renamed.
@bkubiak:
- https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/516031006/Documentation+default+ids
- https://github.com/livechat/api/blob/master/services/agent-api/README.md

**Review by bkubiak — APPROVED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**